spring-commercetools/src/main/java/com/epam/esm/commercetools/repository/GiftCertificateCommerceRepository.java

```diff
@@ -0,0 +1,28 @@
+package com.epam.esm.commercetools.repository;
+
+import com.epam.esm.commercetools.PagePaginationBuilder;
+import com.epam.esm.giftcertficate.model.GiftCertificate;
+
+import java.util.List;
+
+public interface GiftCertificateCommerceRepository<T> {
```

Comment:
Why do you need generics here?

Reply:
At first, I wanted to have one repository for two certificates, but then I realized that it would not work, so I left it up, it would not be possible to remove ?

Reply:
Why not possible to remove? I think you don't need generic here, because you already specify in the name that this is the repository for GiftCertificates.

Reply:
Why not possible to remove? I think you don't need generic here, because you already specify in the name that this is the repository for GiftCertificates.

Reply:
removed unnecessary generics